compiler/src/iree/compiler/Dialect/Flow/Transforms/FoldUnitExtentDims.cpp

```diff
@@ -38,8 +104,35 @@ struct FoldUnitExtentDimsPass
 } // namespace
 
 void FoldUnitExtentDimsPass::runOnOperation() {
-  Operation *funcOp = getOperation();
+  auto moduleOp = getOperation();
   MLIRContext *context = &getContext();
+  Explorer explorer(moduleOp, TraversalAction::RECURSE);
+  explorer.initialize();
```

Comment:
I think you can do the global info initialization on its own right?

Reply:
No, that's a private function. I guess I could make it public, though. @benvanik do you care if I make `Explorer::initializeGlobalInfos` public so I don't have to initialize the inverseCallGraph?